docs/source/gallery/model_comparison/plot_compare.py

```diff
@@ -0,0 +1,27 @@
+"""
+# Predicive model comparison plot
+
+Compare multiple models using predictive accuracy estimates like  {abbr}`LOO-CV (leave one out cross validation)` or {abbr}`WAIC (widely applicable information criterion)`
+
+---
+
+:::{seealso}
+API Documentation: {func}`~arviz_plots.plot_compare`
+:::
```

Comment:
If there are any docs/url for the dashboard version of plot compare we could link there in here too

Reply:
Not now, we need to refactor the entire dashboard project.